src/server/coprocessor/executor/topn.rs

```diff
@@ -29,16 +29,16 @@ use super::{Executor, Row};
 use super::super::Result;
 use super::super::endpoint::{inflate_with_col, SortRow, TopNHeap};
 
-struct ExprColumnRefVisitor {
-    col_ids: HashSet<i64>,
+pub struct ExprColumnRefVisitor {
```

Comment:
Could we move `ExprColumnRefVisitor`  to `src/server/coprocessor/executor/mod.rs` since it would be used in more than one executor?